diff -uNrp ./src/pvrclient-mythtv.cpp ./src/pvrclient-mythtv.cpp
--- ./src/pvrclient-mythtv.cpp	2026-04-23 08:34:46.000000000 +0100
+++ ./src/pvrclient-mythtv.cpp	2026-07-01 08:01:00.403628504 +0100
@@ -252,10 +252,7 @@ PVR_ERROR PVRClientMythTV::GetBackendHos
 
 PVR_ERROR PVRClientMythTV::GetConnectionString(std::string& connection)
 {
-  if (m_control)
-  {
-    connection.assign(m_control->GetBaseURL());
-  }
+  connection.append("http://").append(CMythSettings::GetMythHostname()).append(":").append(Myth::IntToString(CMythSettings::GetWSApiPort()));
   kodi::Log(ADDON_LOG_DEBUG, "%s: %s", __FUNCTION__, connection.c_str());
   return PVR_ERROR_NO_ERROR;
 }
