diff -up ./CMakeLists.txt.orig ./CMakeLists.txt
--- ./CMakeLists.txt.orig	2017-04-19 15:44:27.000000000 +0200
+++ ./CMakeLists.txt	2017-04-24 23:49:24.550567499 +0200
@@ -10,6 +10,13 @@ enable_language(C)
 find_package(Kodi REQUIRED)
 find_package(kodiplatform REQUIRED)
 find_package(p8-platform REQUIRED)
+find_package(PkgConfig)
+pkg_check_modules(CPPMYTH REQUIRED cppmyth)
+
+set(CFLAGS ${CPPMYTH_CFLAGS} ${CPPMYTH_CFLAGS_OTHERS})
+add_definitions(${CFLAGS})
+set(LIBS ${CPPMYTH_CFLAGS} ${CPPMYTH_LIBRARIES})
+link_libraries(${LIBS})
 
 include_directories(${kodiplatform_INCLUDE_DIRS}
                     ${p8-platform_INCLUDE_DIRS}
@@ -51,8 +58,6 @@ set(MYTHTV_SOURCES src/avinfo.cpp
                    src/fileOps.cpp
                    src/filestreaming.cpp)
 
-add_subdirectory(lib/cppmyth)
-
 set(DEPLIBS ${p8-platform_LIBRARIES} cppmyth)
 if(WIN32)
   list(APPEND DEPLIBS ws2_32)
diff -up ./src/avinfo.h.orig ./src/avinfo.h
--- ./src/avinfo.h.orig	2017-04-19 15:44:27.000000000 +0200
+++ ./src/avinfo.h	2017-04-24 23:47:36.928418218 +0200
@@ -24,7 +24,7 @@
 #include "demuxer/tsDemuxer.h"
 #include "client.h"
 
-#include <mythstream.h>
+#include <cppmyth/mythstream.h>
 
 #include <set>
 #include <vector>
diff -up ./src/cppmyth/MythChannel.h.orig ./src/cppmyth/MythChannel.h
--- ./src/cppmyth/MythChannel.h.orig	2017-04-19 15:44:27.000000000 +0200
+++ ./src/cppmyth/MythChannel.h	2017-04-24 23:47:36.929418191 +0200
@@ -19,7 +19,7 @@
  *
  */
 
-#include <mythtypes.h>
+#include <cppmyth/mythtypes.h>
 
 class MythChannel
 {
diff -up ./src/cppmyth/MythEPGInfo.h.orig ./src/cppmyth/MythEPGInfo.h
--- ./src/cppmyth/MythEPGInfo.h.orig	2017-04-19 15:44:27.000000000 +0200
+++ ./src/cppmyth/MythEPGInfo.h	2017-04-24 23:47:36.929418191 +0200
@@ -19,7 +19,7 @@
  *
  */
 
-#include <mythtypes.h>
+#include <cppmyth/mythtypes.h>
 
 class MythEPGInfo
 {
diff -up ./src/cppmyth/MythProgramInfo.h.orig ./src/cppmyth/MythProgramInfo.h
--- ./src/cppmyth/MythProgramInfo.h.orig	2017-04-19 15:44:27.000000000 +0200
+++ ./src/cppmyth/MythProgramInfo.h	2017-04-24 23:47:36.929418191 +0200
@@ -19,7 +19,7 @@
  *
  */
 
-#include <mythtypes.h>
+#include <cppmyth/mythtypes.h>
 
 class MythProgramInfo;
 typedef std::map<std::string, MythProgramInfo> ProgramInfoMap;
diff -up ./src/cppmyth/MythRecordingRule.h.orig ./src/cppmyth/MythRecordingRule.h
--- ./src/cppmyth/MythRecordingRule.h.orig	2017-04-19 15:44:27.000000000 +0200
+++ ./src/cppmyth/MythRecordingRule.h	2017-04-24 23:47:36.930418165 +0200
@@ -19,7 +19,7 @@
  *
  */
 
-#include <mythtypes.h>
+#include <cppmyth/mythtypes.h>
 
 class MythRecordingRule
 {
diff -up ./src/cppmyth/MythScheduleManager.h.orig ./src/cppmyth/MythScheduleManager.h
--- ./src/cppmyth/MythScheduleManager.h.orig	2017-04-19 15:44:27.000000000 +0200
+++ ./src/cppmyth/MythScheduleManager.h	2017-04-24 23:47:36.930418165 +0200
@@ -19,7 +19,7 @@
  *
  */
 
-#include <mythcontrol.h>
+#include <cppmyth/mythcontrol.h>
 #include "MythRecordingRule.h"
 #include "MythProgramInfo.h"
 #include "MythEPGInfo.h"
diff -up ./src/demux.h.orig ./src/demux.h
--- ./src/demux.h.orig	2017-04-19 15:44:27.000000000 +0200
+++ ./src/demux.h	2017-04-24 23:47:36.931418138 +0200
@@ -24,7 +24,7 @@
 #include "demuxer/tsDemuxer.h"
 #include "client.h"
 
-#include <mythstream.h>
+#include <cppmyth/mythstream.h>
 #include <p8-platform/threads/threads.h>
 #include <p8-platform/threads/mutex.h>
 #include <p8-platform/util/buffer.h>
diff -up ./src/fileOps.h.orig ./src/fileOps.h
--- ./src/fileOps.h.orig	2017-04-19 15:44:27.000000000 +0200
+++ ./src/fileOps.h	2017-04-24 23:47:36.931418138 +0200
@@ -24,7 +24,7 @@
 #include "cppmyth/MythChannel.h"
 #include "cppmyth/MythProgramInfo.h"
 
-#include <mythwsapi.h>
+#include <cppmyth/mythwsapi.h>
 #include <p8-platform/threads/threads.h>
 
 #include <string>
diff -up ./src/filestreaming.h.orig ./src/filestreaming.h
--- ./src/filestreaming.h.orig	2017-04-19 15:44:27.000000000 +0200
+++ ./src/filestreaming.h	2017-04-24 23:47:36.931418138 +0200
@@ -21,7 +21,7 @@
  *
  */
 
-#include <mythstream.h>
+#include <cppmyth/mythstream.h>
 
 class FileStreaming : public Myth::Stream
 {
diff -up ./src/pvrclient-mythtv.h.orig ./src/pvrclient-mythtv.h
--- ./src/pvrclient-mythtv.h.orig	2017-04-19 15:44:27.000000000 +0200
+++ ./src/pvrclient-mythtv.h	2017-04-24 23:47:36.932418112 +0200
@@ -29,12 +29,12 @@
 
 #include <xbmc_pvr_types.h>
 #include <p8-platform/threads/mutex.h>
-#include <mythsharedptr.h>
-#include <mythcontrol.h>
-#include <mytheventhandler.h>
-#include <mythlivetvplayback.h>
-#include <mythrecordingplayback.h>
-#include <mythdebug.h>
+#include <cppmyth/mythsharedptr.h>
+#include <cppmyth/mythcontrol.h>
+#include <cppmyth/mytheventhandler.h>
+#include <cppmyth/mythlivetvplayback.h>
+#include <cppmyth/mythrecordingplayback.h>
+#include <cppmyth/mythdebug.h>
 
 #include <string>
 #include <vector>
