diff -up ./CMakeLists.txt.orig ./CMakeLists.txt
--- ./CMakeLists.txt.orig	2020-01-09 19:52:50.000000000 +0100
+++ ./CMakeLists.txt	2020-01-13 14:55:44.178476740 +0100
@@ -7,8 +7,11 @@ set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 find_package(Kodi REQUIRED)
+find_package(PkgConfig)
+pkg_check_modules(CPPMYTH REQUIRED cppmyth)
 
 include_directories(${KODI_INCLUDE_DIR}/.. # Hack way with "/..", need bigger Kodi cmake rework to match right include ways
+                    ${CPPMYTH_INCLUDE_DIRS}
                     ${PROJECT_SOURCE_DIR}/lib/cppmyth/src)
 
 add_definitions(-DUSE_DEMUX -D__STDC_FORMAT_MACROS)
@@ -27,9 +30,7 @@ file (GLOB MYTHTV_HEADERS
   src/cppmyth/*.h
   src/*.h)
 
-add_subdirectory(lib/cppmyth)
-
-set(DEPLIBS cppmyth)
+set(DEPLIBS ${CPPMYTH_LIBRARIES})
 if(WIN32)
   list(APPEND DEPLIBS ws2_32)
 else()
diff -up ./src/artworksmanager.h.orig ./src/artworksmanager.h
--- ./src/artworksmanager.h.orig	2020-01-09 19:52:50.000000000 +0100
+++ ./src/artworksmanager.h	2020-01-13 14:59:21.554803199 +0100
@@ -24,7 +24,7 @@
 #include "cppmyth/MythChannel.h"
 #include "cppmyth/MythProgramInfo.h"
 
-#include <mythwsapi.h>
+#include <cppmyth/mythwsapi.h>
 
 #include <string>
 
diff -up ./src/avinfo.h.orig ./src/avinfo.h
--- ./src/avinfo.h.orig	2020-01-09 19:52:50.000000000 +0100
+++ ./src/avinfo.h	2020-01-13 14:59:21.545803434 +0100
@@ -24,7 +24,7 @@
 #include "demuxer/tsDemuxer.h"
 #include "client.h"
 
-#include <mythstream.h>
+#include <cppmyth/mythstream.h>
 
 #include <set>
 #include <vector>
diff -up ./src/cppmyth/MythChannel.h.orig ./src/cppmyth/MythChannel.h
--- ./src/cppmyth/MythChannel.h.orig	2020-01-09 19:52:50.000000000 +0100
+++ ./src/cppmyth/MythChannel.h	2020-01-13 14:59:21.606801842 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-#include <mythtypes.h>
+#include <cppmyth/mythtypes.h>
 
 class MythChannel
 {
diff -up ./src/cppmyth/MythEPGInfo.h.orig ./src/cppmyth/MythEPGInfo.h
--- ./src/cppmyth/MythEPGInfo.h.orig	2020-01-09 19:52:50.000000000 +0100
+++ ./src/cppmyth/MythEPGInfo.h	2020-01-13 14:59:21.610801738 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-#include <mythtypes.h>
+#include <cppmyth/mythtypes.h>
 
 class MythEPGInfo
 {
diff -up ./src/cppmyth/MythProgramInfo.h.orig ./src/cppmyth/MythProgramInfo.h
--- ./src/cppmyth/MythProgramInfo.h.orig	2020-01-09 19:52:50.000000000 +0100
+++ ./src/cppmyth/MythProgramInfo.h	2020-01-13 14:59:21.628801268 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-#include <mythtypes.h>
+#include <cppmyth/mythtypes.h>
 
 class MythProgramInfo;
 typedef std::map<std::string, MythProgramInfo> ProgramInfoMap;
diff -up ./src/cppmyth/MythRecordingRule.h.orig ./src/cppmyth/MythRecordingRule.h
--- ./src/cppmyth/MythRecordingRule.h.orig	2020-01-09 19:52:50.000000000 +0100
+++ ./src/cppmyth/MythRecordingRule.h	2020-01-13 14:59:21.617801555 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-#include <mythtypes.h>
+#include <cppmyth/mythtypes.h>
 
 class MythRecordingRule
 {
diff -up ./src/cppmyth/MythScheduleManager.h.orig ./src/cppmyth/MythScheduleManager.h
--- ./src/cppmyth/MythScheduleManager.h.orig	2020-01-09 19:52:50.000000000 +0100
+++ ./src/cppmyth/MythScheduleManager.h	2020-01-13 14:59:21.632801164 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-#include <mythcontrol.h>
+#include <cppmyth/mythcontrol.h>
 #include "MythRecordingRule.h"
 #include "MythProgramInfo.h"
 #include "MythEPGInfo.h"
diff -up ./src/filestreaming.h.orig ./src/filestreaming.h
--- ./src/filestreaming.h.orig	2020-01-09 19:52:50.000000000 +0100
+++ ./src/filestreaming.h	2020-01-13 14:59:21.540803565 +0100
@@ -21,7 +21,7 @@
  *
  */
 
-#include <mythstream.h>
+#include <cppmyth/mythstream.h>
 
 class FileStreaming : public Myth::Stream
 {
diff -up ./src/pvrclient-mythtv.h.orig ./src/pvrclient-mythtv.h
--- ./src/pvrclient-mythtv.h.orig	2020-01-09 19:52:50.000000000 +0100
+++ ./src/pvrclient-mythtv.h	2020-01-13 14:59:21.550803304 +0100
@@ -26,12 +26,12 @@
 #include "categories.h"
 
 #include <kodi/xbmc_pvr_types.h>
-#include <mythsharedptr.h>
-#include <mythcontrol.h>
-#include <mytheventhandler.h>
-#include <mythlivetvplayback.h>
-#include <mythrecordingplayback.h>
-#include <mythdebug.h>
+#include <cppmyth/mythsharedptr.h>
+#include <cppmyth/mythcontrol.h>
+#include <cppmyth/mytheventhandler.h>
+#include <cppmyth/mythlivetvplayback.h>
+#include <cppmyth/mythrecordingplayback.h>
+#include <cppmyth/mythdebug.h>
 
 #include <string>
 #include <vector>
