diff -up ./CMakeLists.txt.orig ./CMakeLists.txt
--- ./CMakeLists.txt.orig	2020-07-15 07:57:28.000000000 +0000
+++ ./CMakeLists.txt	2020-08-20 09:30:20.602092660 +0000
@@ -7,8 +7,11 @@ set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 find_package(Kodi REQUIRED)
+find_package(PkgConfig)
+pkg_check_modules(CPPMYTH REQUIRED cppmyth)
 
 include_directories(${KODI_INCLUDE_DIR}/.. # Hack way with "/..", need bigger Kodi cmake rework to match right include ways
+                    ${CPPMYTH_INCLUDE_DIRS}
                     ${PROJECT_SOURCE_DIR}/lib/cppmyth/src)
 
 add_definitions(-DUSE_DEMUX -D__STDC_FORMAT_MACROS)
@@ -27,9 +30,7 @@ file (GLOB MYTHTV_HEADERS
   src/cppmyth/*.h
   src/*.h)
 
-add_subdirectory(lib/cppmyth)
-
-set(DEPLIBS cppmyth)
+set(DEPLIBS ${CPPMYTH_LIBRARIES})
 if(WIN32)
   list(APPEND DEPLIBS ws2_32)
 else()
diff -up ./src/artworksmanager.h.orig ./src/artworksmanager.h
--- ./src/artworksmanager.h.orig	2020-07-15 07:57:28.000000000 +0000
+++ ./src/artworksmanager.h	2020-08-20 09:56:06.292325528 +0000
@@ -25,7 +25,7 @@
 #include "cppmyth/MythProgramInfo.h"
 
 #include <kodi/AddonBase.h>
-#include <mythwsapi.h>
+#include <cppmyth/mythwsapi.h>
 
 #include <string>
 
diff -up ./src/avinfo.h.orig ./src/avinfo.h
--- ./src/avinfo.h.orig	2020-07-15 07:57:28.000000000 +0000
+++ ./src/avinfo.h	2020-08-20 09:56:06.289325602 +0000
@@ -23,7 +23,7 @@
 
 #include "demuxer/tsDemuxer.h"
 
-#include <mythstream.h>
+#include <cppmyth/mythstream.h>
 #include <kodi/AddonBase.h>
 
 #include <set>
diff -up ./src/cppmyth/MythChannel.h.orig ./src/cppmyth/MythChannel.h
--- ./src/cppmyth/MythChannel.h.orig	2020-07-15 07:57:28.000000000 +0000
+++ ./src/cppmyth/MythChannel.h	2020-08-20 09:56:06.270326068 +0000
@@ -19,7 +19,7 @@
  *
  */
 
-#include <mythtypes.h>
+#include <cppmyth/mythtypes.h>
 
 class MythChannel
 {
diff -up ./src/cppmyth/MythEPGInfo.h.orig ./src/cppmyth/MythEPGInfo.h
--- ./src/cppmyth/MythEPGInfo.h.orig	2020-07-15 07:57:28.000000000 +0000
+++ ./src/cppmyth/MythEPGInfo.h	2020-08-20 09:56:06.277325896 +0000
@@ -19,7 +19,7 @@
  *
  */
 
-#include <mythtypes.h>
+#include <cppmyth/mythtypes.h>
 
 class MythEPGInfo
 {
diff -up ./src/cppmyth/MythProgramInfo.h.orig ./src/cppmyth/MythProgramInfo.h
--- ./src/cppmyth/MythProgramInfo.h.orig	2020-07-15 07:57:28.000000000 +0000
+++ ./src/cppmyth/MythProgramInfo.h	2020-08-20 09:56:06.283325749 +0000
@@ -19,7 +19,7 @@
  *
  */
 
-#include <mythtypes.h>
+#include <cppmyth/mythtypes.h>
 
 class MythProgramInfo;
 typedef std::map<std::string, MythProgramInfo> ProgramInfoMap;
diff -up ./src/cppmyth/MythRecordingRule.h.orig ./src/cppmyth/MythRecordingRule.h
--- ./src/cppmyth/MythRecordingRule.h.orig	2020-07-15 07:57:28.000000000 +0000
+++ ./src/cppmyth/MythRecordingRule.h	2020-08-20 09:56:06.274325970 +0000
@@ -19,7 +19,7 @@
  *
  */
 
-#include <mythtypes.h>
+#include <cppmyth/mythtypes.h>
 
 class MythRecordingRule
 {
diff -up ./src/cppmyth/MythScheduleManager.h.orig ./src/cppmyth/MythScheduleManager.h
--- ./src/cppmyth/MythScheduleManager.h.orig	2020-07-15 07:57:28.000000000 +0000
+++ ./src/cppmyth/MythScheduleManager.h	2020-08-20 09:56:06.280325823 +0000
@@ -19,7 +19,7 @@
  *
  */
 
-#include <mythcontrol.h>
+#include <cppmyth/mythcontrol.h>
 #include "MythRecordingRule.h"
 #include "MythProgramInfo.h"
 #include "MythEPGInfo.h"
diff -up ./src/filestreaming.h.orig ./src/filestreaming.h
--- ./src/filestreaming.h.orig	2020-07-15 07:57:28.000000000 +0000
+++ ./src/filestreaming.h	2020-08-20 09:56:06.286325675 +0000
@@ -23,7 +23,7 @@
 
 #include <kodi/Filesystem.h>
 
-#include <mythstream.h>
+#include <cppmyth/mythstream.h>
 
 class ATTRIBUTE_HIDDEN FileStreaming : public Myth::Stream
 {
diff -up ./src/pvrclient-mythtv.h.orig ./src/pvrclient-mythtv.h
--- ./src/pvrclient-mythtv.h.orig	2020-07-15 07:57:28.000000000 +0000
+++ ./src/pvrclient-mythtv.h	2020-08-20 09:56:06.267326142 +0000
@@ -27,12 +27,12 @@
 #include "settings.h"
 
 #include <kodi/addon-instance/PVR.h>
-#include <mythsharedptr.h>
-#include <mythcontrol.h>
-#include <mytheventhandler.h>
-#include <mythlivetvplayback.h>
-#include <mythrecordingplayback.h>
-#include <mythdebug.h>
+#include <cppmyth/mythsharedptr.h>
+#include <cppmyth/mythcontrol.h>
+#include <cppmyth/mytheventhandler.h>
+#include <cppmyth/mythlivetvplayback.h>
+#include <cppmyth/mythrecordingplayback.h>
+#include <cppmyth/mythdebug.h>
 
 #include <string>
 #include <vector>
